.golangci.yaml

```diff
@@ -13,3 +14,7 @@ linters:
 run:
   tests: true
   timeout: 1m
+
+linters-settings:
+  goimports:
+    local-prefixes: "sigs.k8s.io/dra-example-driver"
```

Comment:
This also should go into a separate PR which cleans up the import grouping.

Reply:
Done, I removed that change